tests/test_decoders.py

```diff
@@ -9,6 +9,7 @@
 import zstandard as zstd
 
 import httpx
+from httpx._decoders import GZipDecoder
```

Comment:
Rather than using a private import, could we switch this to use the same style as the other test cases here.

Reply:
I changed the style of the test case